### PR TITLE
feat: translate page with offers

### PR DESCRIPTION
**Pull request by hanny21**

### CODE REVIEW

File: components/OfferSearch.tsx [hunk 1]
```diff
@@ -18,12 +19,16 @@ export type OfferSearchProps = {
 	offerType: Record<string, any>;
 };
 
+const CZECH = "cs";
+
```
Comment (ViliamV) [inline components/OfferSearch.tsx:23]: maybe we should extract these language constants to separate `constants.ts` file

File: components/RegisterForm.tsx [hunk 2]
```diff
@@ -37,7 +39,10 @@ export const RegisterForm = memo<PublicQueryResult>(
 						"Content-Type": "application/json",
 					},
 					body: JSON.stringify({
-						data: state,
+						data: {
+							...state,
+							isUKLanguage: locale === "uk",
```
Comment (ViliamV) [inline components/RegisterForm.tsx:44]: is this used anywhere?

**Review by ViliamV — COMMENTED** [review]
Some minor things

File: components/OfferSearch.tsx (hunk 1 above)
Comment (hanny21) [reply components/OfferSearch.tsx]: yes, you are right.. added `utils` folder and `constants.ts` file

File: components/RegisterForm.tsx (hunk 2 above)
Comment (hanny21) [reply components/RegisterForm.tsx]: tbh not sure.. do you have any idea if we need it elsewhere?

**Review by Michal-Miky-Jankovsky — APPROVED** [review]

**Pull request merged**